packages/node_modules/nav-frontend-skjema/src/textarea.tsx

```diff
@@ -1,20 +1,97 @@
-import PT from 'prop-types';
-import React, { Component } from 'react';
+import * as PT from 'prop-types';
+import * as React from 'react';
 import { EventThrottler, guid, autobind, requestAnimationFrame } from 'nav-frontend-js-utils';
-import 'nav-frontend-skjema-style'; // eslint-disable-line import/extensions
-import classNames from 'classnames';
-import SkjemaelementFeilmelding, { skjemaelementFeilmeldingShape } from './skjemaelement-feilmelding';
+import * as classNames from 'classnames';
+import SkjemaelementFeilmelding, { skjemaelementFeilmeldingShape, SkjemaelementFeil } from './skjemaelement-feilmelding';
+
+import 'nav-frontend-skjema-style';
 
 const inputCls = (className, harFeil) => classNames(
     className,
     'skjemaelement__input textarea--medMeta',
     { 'skjemaelement__input--harFeil': harFeil }
 );
 
+export interface TextareaProps extends React.HTMLAttributes<HTMLTextAreaElement> {
+    label: React.ReactNode;
+    maxLength?: number;
+    value: string;
+    textareaClass?: string;
+    id?: string;
+    name?: string;
+    onChange: (event: React.SyntheticEvent<EventTarget>) => void;
+    feil?: SkjemaelementFeil;
+    tellerTekst?: (antallTegn: number, maxLength: number) => React.ReactNode;
+    textareaRef?: () => any;
+}
+
+interface TextareaDefaultProps {
+    maxLength: number;
+    textareaClass: string;
+    tellerTekst: (antallTegn: number, maxLength: number) => React.ReactNode;
+}
+
+type PropsWithDefault = TextareaProps & TextareaDefaultProps;
+
 /**
  * Selvekspanderende tekstområde med teller
  */
-class Textarea extends Component {
+class Textarea extends React.Component<TextareaProps> {
```

Comment:
Får `Illegal invocation` runtime feil på linje 102 under `componentDidMount` her: 
```JSX
requestAnimationFrame(this.updateHeight, 0);
``` 
Litt rart da du ikke har endret noe på denne, eller i `js-utils` hvor den henter metoden fra 🤔

Reply:
Hvordan er det du tester? Jeg kjørte `npm run start-guildeline-app` og får ingen feil der. Var etter jeg gjorde de siste endringene jeg nettopp pushet.

Reply:
Inkluderer en enkel `<TextareaControlled />` i dev-miljø (`npm run dev`). Får forøvrig samme feil i Guideline-appen også. Har pullet siste oppdateringen din og får fortsatt samme feil begge steder.

Opera/Chrome:
<img width="530" alt="screen shot 2018-04-23 at 12 53 16" src="https://user-images.githubusercontent.com/74510/39122508-6f2de048-46f5-11e8-85a3-5626556320dd.png">

Firefox:
<img width="681" alt="screen shot 2018-04-23 at 12 53 49" src="https://user-images.githubusercontent.com/74510/39122518-761ab3e0-46f5-11e8-9014-b31e3022cc68.png">

Reply:
Ser den kan fikses ved å endre til:
```JSX
requestAnimationFrame.call(window, this.updateHeight, 0);
```
Men litt rart hvis det bare er jeg som får den feilen.

Reply:
Fikk testet og fikset det nå 😄